Applications/Utils/OGSFileConverter/OGSFileConverter.cpp

```diff
@@ -45,15 +45,15 @@ void OGSFileConverter::convertGML2GLI(const QStringList &input, const QString &o
     GeoLib::GEOObjects geo_objects;
     GeoLib::IO::XmlGmlInterface xml(geo_objects);
 
-    for (const auto& it : input)
+    for (const auto& string : input)
```

Comment:
Very minor: `string` is very similar to `std::string`. Is it a good idea to use it as a veriable name?